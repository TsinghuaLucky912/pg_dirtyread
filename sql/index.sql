select setting::int >= 160000 as is_pg16 from pg_settings where name = 'server_version_num';

CREATE INDEX ON foo(bar);
SELECT * FROM pg_dirtyread('foo_bar_idx') as t(bar bigint);
