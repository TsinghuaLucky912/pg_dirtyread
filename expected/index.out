select setting::int >= 160000 as is_pg16 from pg_settings where name = 'server_version_num';
 is_pg16 
---------
 f
(1 row)

CREATE INDEX ON foo(bar);
SELECT * FROM pg_dirtyread('foo_bar_idx') as t(bar bigint);
ERROR:  "foo_bar_idx" is an index
